graphistry/embed_utils.py

```diff
@@ -153,13 +154,13 @@ def _build_graph(self, res) -> Plottable:
             g_dgl.edata[dgl.ETYPE] = r
 
         g_dgl.edata["norm"] = dgl.norm_by_dst(g_dgl).unsqueeze(-1)
-        res.g_dgl = g_dgl
+        res._kg_dgl = g_dgl
```

Comment:
I am thinking we might not need `_kg_dgl` as we have `DGL_graph` because both are going to output a dglgraph

Reply:
they are two totally different graphs -- `DGL_graph` is a general graph object for use in pipelines, and `_kg_dgl` is for the knowledge graph embedding. They should have different namespace.